src/NuGet.Protocol.Catalog/Models/AlternatePackage.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Newtonsoft.Json;
+
+namespace NuGet.Protocol.Catalog
+{
+    public class AlternatePackage
```

Comment:
Please add a link to the documentation. Also, it seems the documentation needs to be updated to describe the shape of the `alternatePackage` object

Reply:
It's defined as "The package dependency that should be used instead", where "package dependency" is an object that's defined above. Perhaps it should have a link to the package dependency definition but I believe it is technically sufficient.

Reply:
Ah. That's not evident, I would definitely include a link.

Reply:
https://github.com/NuGet/docs.microsoft.com-nuget/pull/1766

Reply:
Done.